sechub-pds-solutions/findsecuritybugs/tests/download_test_files.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+current_test_folder="$1"
+test_file="$2"
+test_folder_name="$3"
+	
+if [[ ! -d "$current_test_folder" ]]
+then
+	echo "Target folder is empty"
+	exit 1
+fi
+
+if [[ -z "$test_file" ]]
+then
+	echo "No file to test provided"
+	exit 1
+fi
+
+if [[ -z "$test_folder_name" ]]
+then
+	echo "No file to test provided"
```

Comment:
Same text as in line 16. Is this correct?
The variable name sounds like a folder and not like a file.